code/drasil-docLang/lib/Drasil/DocumentLanguage/Definitions.hs

```diff
@@ -95,16 +98,16 @@ tmDispExprs t = map express (t ^. defined_quant) ++ t ^. invariants
 
 -- | Create the fields for a model from a relation concept (used by 'tmodel').
 mkTMField :: TheoryModel -> SystemInformation -> Field -> ModRow -> ModRow
-mkTMField t _ l@Label fs  = (show l, [mkParagraph $ atStart t]) : fs
-mkTMField t _ l@DefiningEquation fs = (show l, map unlbldExpr $ tmDispExprs t) : fs
-mkTMField t m l@(Description v u) fs = (show l,
+mkTMField t _ l@Label fs  = (getSent l, [mkParagraph $ atStart t]) : fs
+mkTMField t _ l@DefiningEquation fs = (getSent l, map unlbldExpr $ tmDispExprs t) : fs
+mkTMField t m l@(Description v u) fs = (getSent l,
   foldr ((\x -> buildDescription v u x m) . express) [] $ tmDispExprs t) : fs
-mkTMField t m l@RefBy fs = (show l, [mkParagraph $ helperRefs t m]) : fs --FIXME: fill this in
-mkTMField t _ l@Source fs = (show l, helperSources $ t ^. getDecRefs) : fs
+mkTMField t m l@RefBy fs = (getSent l, [mkParagraph $ helperRefs t m]) : fs --FIXME: fill this in
+mkTMField t _ l@Source fs = (getSent l, helperSources $ t ^. getDecRefs) : fs
 mkTMField t _ l@Notes fs =
-  nonEmpty fs (\ss -> (show l, map mkParagraph ss) : fs) (t ^. getNotes)
-mkTMField _ _ l _ = error $ "Label " ++ show l ++ " not supported " ++
-  "for theory models"
+  nonEmpty fs (\ss -> (getSent l, map mkParagraph ss) : fs) (t ^. getNotes)
+mkTMField _ _ l fs = (getSent l +:+ S ":Label not supported for data definitions", []) : fs
```

Comment:
a) This should still say "theory models" since it is the `mkTMField` constructor
b) I think the error message should stay as it is; if a pattern above isn't matched, then what value of `l` would be possible that would be caught by a case of `getSent`?

(This applies to the other instances as well)

Reply:
in this specific case I was thinking something like input constraints, that is a field that could be passed but is not supported by mkTMField

Reply:
Yes, you are right - my bad! I don't know why I just assumed that there would be a case for every label in these field functions 😔